live-examples/js-examples/dataview/dataview-getbigint64.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
+// Highest possible BigInt value that fits in a signed 64-bit integer
+const max = 2n ** (64n -1n) - 1n;
```

Comment:
It's odd to mix `var` and `const` like this. Can we use `const` throughout?

Reply:
Agreed. This was me just copy-pasting from the existing samples and not thinking. Good catch! :)